docs/deployment_guide.md

```diff
@@ -44,12 +44,22 @@ Please see [HERE for details](https://docs.aws.amazon.com/cli/latest/userguide/c
 
 #### Bootstrap the CDKV2
 
-We use AWS CDK V2 as the standard CDK version because CDK V1 is set to maintenance mode beginning June 1, 2022. You  would need to bootstrap the CDK environment (one time per region) with V2 - see [HERE](https://docs.aws.amazon.com/cdk/v2/guide/bootstrapping.html).
+We use AWS CDK V2 as the standard CDK version because CDK V1 is set to maintenance mode beginning June 1, 2022. You  would need to bootstrap the CDK environment (one time per region) with V2 - see [HERE](https://docs.aws.amazon.com/cdk/v2/guide/bootstrapping.html) in the account(s) you will be deploying.
 
 ```bash
 cdk bootstrap aws://<account>/<region>
 ```
 
+#### Bootstrap AWS Account(s)
+
+Assuming that you will be using a single account, follow the guide [here](https://seed-farmer.readthedocs.io/en/latest/bootstrapping.html#) to bootstrap your account(s) to function as a toolchain and target account.
+
+Following is the command to bootstrap your existing account to a toolchain and target account
+
+```sh
+seedfarmer bootstrap toolchain --project [project] --trusted-principals [principal-arns] --as-target
```

Comment:
since this is the ADDF docs, let's go ahead and set the project on this example to `--project addf`